app/assets/javascripts/tasks_ratings.coffee

```diff
@@ -0,0 +1,74 @@
+ready =->
+  initializeRatings()
+
+$(document).on('turbolinks:load', ready)
+
+initializeRatings = ->
+  USERRATING = 0
+
+#  $('.popup-rating').hover (->
+#    USERRATING = $('.rating span.fa-star').last().attr("data-rating")
+#  ), ->
+#    lower = $('.rating span').filter (->
+#      $(this).attr("data-rating") <= USERRATING
+#    )
+#    $(lower).removeClass("fa-star-o").addClass("fa-star")
+#    upper = $('.rating span').filter (->
+#      $(this).attr("data-rating") > USERRATING
+#    )
+#    $(upper).removeClass("fa-star").addClass("fa-star-o")
```

Comment:
this was in the old code - do you think it serves any purpose?
it seems to work fine without

Reply:
No, not really. It seems duplicated compared to the other logic below, and I would also opt to remove it. If you do so, please also remove the `USERRATING` variable in this file -- it's not used as well, is it?